docs/overview.rst

```diff
@@ -720,6 +720,9 @@ https://copr.fedorainfracloud.org/coprs/g/teemtee/tmt/
 PIP:
 https://pypi.org/project/tmt/
 
+Quay:
+https://quay.io/repository/teemtee/tmt
```

Comment:
```suggestion
https://quay.io/organization/teemtee
```

Or we could link the organization instead?

Reply:
+1

Reply:
Od, done.